### PR TITLE
Make timeout for integration tests configurable 

### PR DESCRIPTION
**Pull request by ejegrova**
Variable timeout is added to test_env_vars.yaml and used in integration tests.

### CODE REVIEW

**Comment by ejegrova** [comment]
> Nice!
> 
> Note that the same commit as the one in #211 is applied here as well.
> 
> I believe we should either have this new patch for configurable timeouts in that same PR (#211) or remove the npm integration tests commit from this PR.

I know there's the same commit, but I don't know how to properly deal with that. Chages in test_npm_basic depends on that commit. Maybe I should just put it in draft and rebase when the first commit is merged.

**Comment by athos-ribeiro** [comment]
> > Nice!
> > Note that the same commit as the one in #211 is applied here as well.
> > I believe we should either have this new patch for configurable timeouts in that same PR (#211) or remove the npm integration tests commit from this PR.
> 
> I know there's the same commit, but I don't know how to properly deal with that. Chages in test_npm_basic depends on that commit. Maybe I should just put it in draft and rebase when the first commit is merged.

Sounds good.

I wouldn't mind having this commit in that PR as well. The code looks good and the re-factoring would make sense. Whatever works for you :)

File: tests/integration/conftest.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def default_request(test_env):
     :return: a dict of packages with initial and completed responses from the Cachito API
     :rtype: dict
     """
-    client = utils.Client(test_env["api_url"], test_env["api_auth_type"])
+    client = utils.Client(test_env["api_url"], test_env["api_auth_type"], test_env["timeout"])
```
Comment (athos-ribeiro) [inline tests/integration/conftest.py:37]: Won't this break current deployments without a timeout configured? Could we use `test_env.get("timeout)` instead?

File: tests/integration/utils.py [hunk 2]
```diff
@@ -14,15 +14,17 @@
 class Client:
     """Simplifies access to the Cachito API."""
 
-    def __init__(self, cachito_api_url, cachito_api_auth_type):
+    def __init__(self, cachito_api_url, cachito_api_auth_type, timeout):
```
Comment (athos-ribeiro) [inline tests/integration/utils.py:17]: Then here we could have timeout=SOME_DEFAULT_VALUE

**Comment by ejegrova** [comment]
made the changes, @athos-ribeiro, can you check please?

**Pull request merged**